src/App.js

```diff
@@ -44,27 +46,52 @@ function App() {
 
   return (
     <div className="App">
-      <header className="App-header">Sleep Tracker</header>
-      {savedGrid.map((day, i) => {
-        return <DayView key={i} row={day} />;
-      })}
-      <Day row={grid} mode={mode} changeRow={setGrid} />
-      <button onClick={clearGrid}>Clear</button>
-      <button onClick={saveGrid}>Save</button>
-      <div>
-        <input
-          onChange={handleModeChange}
-          id="sleep"
-          type="radio"
-          value="sleep"
-          name="mode"
-          defaultChecked
-        />
-        <label htmlFor="sleep">Sleep</label>
-        <input onChange={handleModeChange} id="awake" type="radio" value="awake" name="mode" />
-        <label htmlFor="awake">Awake</label>
-        <input onChange={handleModeChange} id="active" type="radio" value="active" name="mode" />
-        <label htmlFor="active">Active</label>
+      <div className="main-container">
+        <header className="section-container">
+          <h1> Sleep Tracker </h1>
+        </header>
+        <div className="days-container">
+          {savedGrid.map((day, i) => {
+            return <DayView key={i} row={day} />;
+          })}
+        </div>
+
+        <div>
+          <Day row={grid} mode={mode} changeRow={setGrid} />
+        </div>
+
+        <div className="section-container">
+          <input onChange={handleModeChange} id="active" type="radio" value="active" name="mode" />
+          <label htmlFor="active">Active</label>
+
+          <input
+            onChange={handleModeChange}
+            id="falling-asleep"
+            type="radio"
+            value="falling-asleep"
+            name="mode"
+          />
+          <label htmlFor="falling-asleep">Falling asleep</label>
+          <input
+            onChange={handleModeChange}
+            id="sleep"
+            type="radio"
+            value="sleep"
+            name="mode"
+            defaultChecked
+          />
+          <label htmlFor="sleep">Sleep</label>
+        </div>
+        <div className="section-container">
+          <DatePicker
```

Comment:
Why did you choose to use a library? Did you know you could do `<input type="date" />`? :)